meson.build

```diff
@@ -447,7 +447,7 @@ if get_option('build_backends')
       )
 	 files += cuda_files
     files += cuda_gen.process(cuda_files_nvcc_common)
-    files += cuda_gen.process(cuda_files_nvcc_fp16, extra_args: ['-arch=compute_70', '-code=sm_70'])
+    files += cuda_gen.process(cuda_files_nvcc_fp16, extra_args: ['-arch=compute_75', '-code=sm_80', '-code=sm_75', '-code=sm_86'])
```

Comment:
This fails to build with older cuda versions. I have a hack for it in <https://github.com/borg323/lc0/commit/8b359dae61ed228ab580ee4f6f2751efb5dfea97>, but increases binary size a lot, need to check if the target list can be reduced. Alternatively, maybe a simple `extra_args: '-arch=compute_53'`(or whatever) is enough (but want to test if there are any performance implications).

Reply:
Thanks! extra_args: '-arch=compute_53' seems to work without perf degradation.